platform/linux-generic/dumpconfig/dumpconfig.c

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include <odp_libconfig_config.h>
+
+int main(void)
+{
+	unsigned int i;
+
+	for (i = 0; i < sizeof(config_builtin); i++)
+		printf("%c", config_builtin[i]);
```

Comment:
I'm not sure what this utility does. Is it just printing out the static config file found in `config/odp-linux-generic.conf`?  That doesn't seem terribly useful. A utility that printed out the effective configuration (taking the `ODP_CONFIG_FILE` into account) would be more useful, but this doesn't seem to do that.

Reply:
@Bill-Fischofer-Linaro yes odp apps can run without config. Then builtin config is used. And ODP_CONFIG_FILE can just override parts of default config.